tests/integ/test_dataframe.py

```diff
@@ -2126,33 +2122,30 @@ def test_fillna(session):
     Utils.check_answer(
         session.create_dataframe(
             [[1, 1.1], [None, None]], schema=["col1", "col2"]
-        ).fillna({"col1": 1.1, "col2": 1}),
-        [Row(1, 1.1), Row(None, 1)],
+        ).fillna({"col1": 1.1, "col2": 1.1}),
+        [Row(1, 1.1), Row(None, 1.1)],
```

Comment:
what makes we change to `1.1`, does `1` not work any more?

Reply:
Ditto.